### DISCUSSION TITLE
Not sure why I'm having trouble with this, but ...

### DISCUSSION THREAD

**Discussion by FrankNJohnson** [brianpetro/obsidian-smart-connections#489] [Q&A]
A large number of my notes have two sections at the bottom that look like this:

```
## METADATA
***
Source:: 
Domain(s):: 
Note Type:: 
Note Status:: 
Related Notes:: 
Date Created:: 


## OUTPUT
***
Content Idea:: 
Content Status:: 
Content Outlet:: 
```

If I want all of that excluded, what do I put in the Heading Exclusions field? I've tried "## METADATA,## OUTPUT" and "METADATA,OUTPUT" but in both cases, I'm still seeing notes in SmartView that seem to be keying on METADATA. For example, one of the notes in the SmartView pane has this as the note path and title:

0.58 | 1 SEEDBOX > Joshua Barron On Virtue > Joshua Barron On Virtue > METADATA

(I'm not sure why "Joshua Barron On Virtue" is appearing twice, but I guess it could because one is the file name and one is the first H1 heading in the note).

Could the underlines under the H2 headings be a problem? Or are the only headings that can be excluded H1 headings (in which case, I would need to preface the above code with an H1 heading - or change the H2 headings above into H1 headings - I did see an earlier discussion that might suggest that, in which the person mentioned they solved their problem by adding a heading above other headings).

Thanks in advance for any help anyone can offer - it's much appreciated!

**Comment by brianpetro** [comment]
@FrankNJohnson, thanks for bringing this to my attention!

There are a few possibilities here.

1) It could be a bug.

2) It could also be that old embeddings are being used even after you updated the header exclusion settings.

To rule out 2, try running a force refresh after ensuring the header exclusion settings are set correctly. If you have a large vault, you can also use the folder/file exclusion settings to reduce the number of notes used to complete this experiment.

If it's still happening after a force refresh, I'll investigate what bug might be causing the issue.

Thanks for your help in solving this!

🌴

**Reply by FrankNJohnson** [reply]
Thanks @brianpetro . Can you tell me exactly what I need to enter in the Heading Exclusions field to exclude the above blocks? That's where I'm having my first problem - it may just be my lack of familiarity with all things Obsidian, but I can't figure it out and I'm not seeing any specific examples anywhere which show that detail. I've done multiple force refreshes, but I'm thinking they probably wouldn't do any good if I haven't entered the correct thing in the Heading Exclusions field.

**Reply by brianpetro** [reply]
`METADATA,OUTPUT` should be sufficient 🌴

**Reply by FrankNJohnson** [reply]
Hmmmm ... after changing the contents of the Heading Exclusions field back to the original METADATA,OUPUT value and then refreshing notes, there definitely seems to have been a change. Now on that "Joshua Barron on Virtue" listing in the SmartView pane, it no longer shows the connection to the METADATA block, but now shows a connection to the OUTPUT block (which I don't recall seeing before in the note title in the SmartView pane, although it might have still been in the contents of the note shown in that pane. Now when I expand that SmartView pane listing, I see this: 

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/116246915/8548d8d4-85a0-4201-a1b7-099d062ead0f)

Note that there is definitely more than just those Content Idea, Content Status, and Content Outline lines in the actual note. 

I don't know if this will give you any clues or not, but I'm seeing three different patterns that are reflected below:

1. Note 1 (the Joshua Barron on Virtue note) - has this structure:

H1 heading
Textual content
METADATA BLOCK
OUTPUT BLOCK

In the SmartView pane, when Note 1 is expanded, I see the note title and the three content lines I referenced above (but not the CONTENT H2 heading)

2. Note 2 has this structure:
H1 heading
H2 heading
Textual content
METADATA BLOCK
OUTPUT BLOCK

In the SmartView pane, when Note 2 is expanded, I see the note title, the first H2 heading (but not the first H1 heading) and the beginning of the Textual content.

3. Note 3 has this structure:
H1 heading
Textual content
METADATA BLOCK
OUTPUT BLOCK

In the SmartView pane, when Note 3 is expanded, I see the note title and the six content lines that are part of the METADATA block.

So it seems to be all over the place.

As I was installing the plugin (and I had installed it once before and subsequently removed it, but reinstalled it recently) this time, I have changed the settings numerous times (including which notes I wanted it to index - through the various exclusion settings), refreshed the notes numerous times. Does it make sense (and is it possible) to just start over - uninstall the plugin, make sure any folders it had created are removed, and then reinstall the plugin again - and establish the settings one time exactly how I want them.

My instinct is telling me that perhaps all of my changes and refreshes have messed up the system, but that may be a "false instinct."

Thanks for your help with this! It's very much appreciated. Now that I've installed it again - and have gotten a better idea of its promise - I REALLY want to get this to work. :)
